lib/ProxySQL_Admin.cpp

```diff
@@ -98,15 +98,16 @@ pthread_mutex_t admin_mutex = PTHREAD_MUTEX_INITIALIZER;
 
 
 static char * admin_variables_names[]= {
-  (char *)"admin_credentials",
-  (char *)"stats_credentials",
-  (char *)"mysql_ifaces",
-  (char *)"telnet_admin_ifaces",
-  (char *)"telnet_stats_ifaces",
-  (char *)"mysql_ifaces",
-  (char *)"refresh_interval",
+	(char *)"admin_credentials",
+	(char *)"stats_credentials",
+	(char *)"mysql_ifaces",
+	(char *)"telnet_admin_ifaces",
+	(char *)"telnet_stats_ifaces",
+	(char *)"mysql_ifaces",
+	(char *)"refresh_interval",
 	(char *)"read_only",
 	(char *)"version",
+	(char *)"proxysql_consul_path",
```

Comment:
`proxysql` token in the name doesn't bring any value here (I know it comes from how the middleman is named, but that's another problem). Also, don't forget to add this variable to the documentation! :D